iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -503,6 +506,13 @@ struct TileAndDistributeOnTensorsPattern
     SmallVector<Value, 4> count = llvm::to_vector<4>(
         llvm::map_range(linalgOp.createLoopRanges(rewriter, loc),
                         [](Range r) { return r.size; }));
+    // NOTE: Special treatment for convolution, which have more than 3 parallel
```

Comment:
Instead of NOTE: please file a bug and assign it to me. There needs to be a cleaner way of doing this which can decide this just by the tile size. Setting the tile size to zero must automatically make everything make the right choice.

Reply:
Created #5048 to track this. IMHO, we can actually relax to tile along all parallel dimensions when creating `flow.dispatch.workgroups` and then revert unused dimensions by turning the workgroup count to 1. But I understand there are other implications. But given my original prototype was doing that, I'll just send out a pull request later and we can see whether it's acceptable.